app/js/services/situationService.js

```diff
@@ -78,7 +78,9 @@ angular.module('ddsApp').factory('SituationService', function($http, $sessionSto
         },
 
         restoreRemote: function(situationId) {
-            return $http.get('/api/situations/' + situationId).then(function(result) {
+            return $http.get('/api/situations/' + situationId, {
+                params: { cacheBust: Date.now() }
```

Comment:
So if I understand well you add a `?cacheBust=timestamp` to your `GET` request so that the browser think it's a new page and therefore don't use the cached version. And the name `cacheBust` is just used to make the intention clear, but the name of the parameter itself doesn't technically matter. Am I right ?